diff --git a/dist/index.js b/dist/index.js
index 753b7b2..7a053e9 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -1598,7 +1598,7 @@ var SandboxApi = class {
     const client = new ApiClient(config);
     const res = await client.api.POST("/sandboxes", {
       body: {
-        autoPause: false,
+        autoPause: opts.autoPause ?? false,
         templateID: template,
         metadata: opts == null ? void 0 : opts.metadata,
         envVars: opts == null ? void 0 : opts.envs,
diff --git a/dist/index.mjs b/dist/index.mjs
index 457ba46..9197f99 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -1563,7 +1563,7 @@ var SandboxApi = class {
     const client = new ApiClient(config);
     const res = await client.api.POST("/sandboxes", {
       body: {
-        autoPause: false,
+        autoPause: opts.autoPause ?? false,
         templateID: template,
         metadata: opts == null ? void 0 : opts.metadata,
         envVars: opts == null ? void 0 : opts.envs,
